README.md

```diff
@@ -5,4 +5,13 @@ easy-schema-examples
 A collection of examples of the XML schema's used by EASY. These schema's are published at 
 [https://easy.dans.knaw.nl/schemas/](https://easy.dans.knaw.nl/schemas/)
 
+Development cycle
+-----------------
 
+* Add new version of XSD to https://github.com/DANS-KNAW/easy-schema.
+* Run `mvn clean install` in `easy-schema`.
+* Run `mvn clean install generate-test-resources ` in this project.
```

Comment:
with the change from `mvncist` to `mvn clean install generate-test-resources` you are now in trouble, because it runs the tests, which is not what you want in this stage. Use `mvn clean generate-test-resources` (without the `install`) instead.

```suggestion
* Run `mvn clean generate-test-resources` in this project.
```